Translate content/pt/auto_scaling.md to portuguese (#443)
cncf/glossary: content/pt-br/abstraction.md

---
title: Abstração
status: Completed
category: propriedade
---

> No contexto computacional, uma abstração é uma representação que oculta especificidades para um consumidor de serviços, tornando sua utilização mais genérica e de fácil entendimento. Um bom exemplo é o sistema operacional(S.O) do seu laptop. Ele abstrai todos os detalhes de como o computador funciona. Você não precisa ter nenhum conhecimento sobre CPU, memória e como os programas são executados, você apenas opera o S.O e o S.O lida com os detalhes. Todos esses detalhes são ocultos por trás das cortinas do S.O ou seja, uma abstração. 

Reviewer: ```suggestion
No contexto computacional, uma abstração é uma representação que oculta especificidades para um consumidor de serviços, tornando sua utilização mais genérica e de fácil entendimento. Um bom exemplo é o sistema operacional (S.O) do seu laptop. Ele abstrai todos os detalhes de como o computador funciona. Você não precisa ter nenhum conhecimento sobre CPU, memória e como os programas são executados, você apenas opera o S.O e o S.O lida com os detalhes. Todos esses detalhes são ocultos por trás das cortinas do S.O - ou seja, uma abstração. 
```

> Normalmente os sistemas tem multiplas camadas de abstração. Isso simplifica de forma significativa o desenvolvimento. Na programação, os desenvolvedorem constroem componentes compatíveis com uma camada de abstração específica e não precisam se preocupar com todas as especificidades subjacentes, que podem ser heterogênias. Se o componente funcionar com a camada de abstração, ele funciona no sistema - não importando o que está por debaixo dos panos.

Reviewer: ```suggestion
Normalmente os sistemas tem múltiplas camadas de abstração. Isso simplifica de forma significativa o desenvolvimento. Na programação, os desenvolvedorem constroem componentes compatíveis com uma camada de abstração específica e não precisam se preocupar com todas as especificidades subjacentes, que podem ser heterogênias. Se o componente funcionar com a camada de abstração, ele funciona no sistema - não importando o que está por debaixo dos panos.
```

> Normalmente os sistemas tem multiplas camadas de abstração. Isso simplifica de forma significativa o desenvolvimento. Na programação, os desenvolvedorem constroem componentes compatíveis com uma camada de abstração específica e não precisam se preocupar com todas as especificidades subjacentes, que podem ser heterogênias. Se o componente funcionar com a camada de abstração, ele funciona no sistema - não importando o que está por debaixo dos panos.

Reviewer: ```suggestion
Normalmente os sistemas tem multiplas camadas de abstração. Isso simplifica de forma significativa o desenvolvimento. Na programação, os desenvolvedores constroem componentes compatíveis com uma camada de abstração específica e não precisam se preocupar com todas as especificidades subjacentes, que podem ser heterogêneas. Se o componente funcionar com a camada de abstração, ele funciona no sistema - não importando o que está por debaixo dos panos.
```
